docs/FAQ

```diff
@@ -1318,7 +1319,7 @@ FAQ
   you want to change name resolver function you must rebuild libcurl and tell
   it to use a different function.
 
-  - The non-IPv6 resolver that can use one out of four host name resolve calls
+  - The non-IPv6 resolver that can use one out of four host names resolves calls
```

Comment:
I thought the the essence of the sentence is: "The  resolver resolves calls". 
Does the resolver use "one out of four **host names**" or "one out of four **host name resolve calls**"? Is "host name resolve call" a thing?